### ISSUE TITLE
Fix broken login

### ISSUE THREAD

**Issue by mariuszskon** [mariuszskon/gummiebot#1]
Currently, getting the login page (and performing a login) is broken, meaning gummiebot is rather useless.
The issue as I see it, is that Gumtree seems to return a weird mostly-javascript response when making a GET request to the login page. The login page is now `t-login-page.html` but `t-login.html` seems to nicely redirect there.

```
$ curl https://www.gumtree.com.au/t-login-form.html
<html><head><meta charset="utf-8"><script>function i700(){}i700.F20=function (){return typeof i700.O20.p60==='function'?i700.O20.p60.apply(i700.O20,arguments):i700.O20.p60;};i700.X70=function (){
...
storage:"3"};winsocks();</script></head><body></body></html>
```

Seems related to https://stackoverflow.com/questions/58712157/puppeteer-js-return-only-html-header-while-full-html-can-be-seen-in-chromes-dev

**Comment by camrail** [comment]
Hey Mariusz thanks for the repo!  I take it this is still broken?  I'm getting an add type field error and when I inspect the page it looks like I'm not logged in ... so guessing this is probably a symptom of this problem above?  Thank you.

**Comment by mariuszskon** [comment]
Yeah this project is abandoned :cry: so you could be getting many errors.
